src/components/Garden/ModalFlows/ModalFlowBase.tsx

```diff
@@ -14,32 +12,50 @@ import { useActivity } from '@providers/ActivityProvider'
 import { useWallet } from '@providers/Wallet'
 
 import { setAccountSetting } from '@/local-settings'
+import { TransactionType } from '@/hooks/constants'
+
+const indexNumber = ['First', 'Second', 'Third', 'Fourth', 'Fifth']
```

Comment:
since is a constant lets take the opportunity to use uppercase on the const name

Reply:
Fixed already :)